recipes/grpc/all/conanfile.py

```diff
@@ -1,20 +1,24 @@
-import shutil
-from conan import tools
+from conan import ConanFile
+from conan.tools.apple import is_apple_os
+from conan.tools.microsoft import visual, is_msvc
+from conan.tools.build import cross_building, valid_min_cppstd, check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, rmdir, rename, replace_in_file
 from conan.tools.scm import Version
-from conans import ConanFile, CMake, tools as tools_legacy
-from conans.errors import ConanInvalidConfiguration
+from conan.errors import ConanInvalidConfiguration
+from conans import CMake
 import os
+import shutil
 
 required_conan_version = ">=1.49.0"
 
 
 class grpcConan(ConanFile):
     name = "grpc"
     description = "Google's RPC (remote procedure call) library and framework."
-    topics = ("grpc", "rpc")
+    license = "Apache-2.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/grpc/grpc"
-    license = "Apache-2.0"
+    topics = ("rpc")
```

Comment:
stealing words from their description

```suggestion
    topics = ("rpc", "remote procedure call", "application communication", "connected systems")
```